web/src/css/custom.css

```diff
@@ -105,16 +109,16 @@
     no-repeat;
 }
 
-.docusaurus-mt-lg {
-  background-color: #fff;
+[data-theme="dark"] .card {
+  background-color: #0a1120;
```

Comment:
What is affected by this?

Reply:
It changes the pagination card colors @petar-cvit .